src/validations/rules/sar.sch

```diff
@@ -178,47 +242,91 @@
 
     <sch:pattern
         id="results">
-        <sch:let
-            name="ssp-import-url"
-            value="resolve-uri($sap-doc/oscal:assessment-plan/oscal:import-ssp/@href, base-uri())" />
-        <sch:let
-            name="sap-import-url"
-            value="resolve-uri(/oscal:assessment-results/oscal:import-ap/@href, base-uri())" />
-        <sch:let
-            name="sap-available"
-            value="
-                if (: this is not a relative reference :) (not(starts-with($sap-import-url, '#')))
-                then
-                    (: the referenced document must be available :)
-                    doc-available($sap-import-url)
-                else
-                    true()" />
-        <sch:let
-            name="sap-doc"
-            value="
-                if ($sap-available)
-                then
-                    doc($sap-import-url)
-                else
-                    ()" />
-        <sch:let
-            name="ssp-available"
-            value="
-                if (: this is not a relative reference :) (not(starts-with($ssp-import-url, '#')))
-                then
-                    (: the referenced document must be available :)
-                    doc-available($ssp-import-url)
-                else
-                    true()" />
-        <sch:let
-            name="ssp-doc"
-            value="
-                if ($ssp-available)
-                then
-                    doc($ssp-import-url)
-                else
-                    ()" />
+        
+        <sch:rule
+            context="oscal:result">
+            <!-- UUID of most recent result -->
+            <sch:let
+                name="first-result"
+                value="//oscal:result[xs:dateTime(oscal:start) eq max(//oscal:result/oscal:start ! xs:dateTime(.))]/@uuid" />
```

Comment:
Why should the subsequent assertions apply solely to the _most recent_ `result`?

Reply:
I thought that the most recent would be the result of interest.